pkg/client/auth/oidc/pkce.go

```diff
@@ -89,10 +89,20 @@ func AuthenticatePkce(config PKCEDetails) (*TokenCredentials, error) {
 
 	server := &http.Server{}
 
-	go server.Serve(listener)
+	go func() {
+		err := server.Serve(listener)
+		if err != nil {
+			panic(err)
+		}
+	}()
 
 	cmd, err := openBrowser("http://" + localUrl)
-	defer cmd.Process.Kill()
+	defer func() {
+		err := cmd.Process.Kill()
```

Comment:
What should we do if we can't kill the browser that does oidc sign in?

Reply:
we didn't do anything before, so log and move on seems fine

Reply:
Should these be log.Warn or log.Error rather than Printf?

Reply:
`logging.WithStacktrace.Error`